src/components/chat/insurance-table-rows/InsuranceTableRows.tsx

```diff
@@ -3,44 +3,48 @@ import { ACTIVATION_DATE, CANCELLATION_DATE } from 'lib/messageTypes'
 import * as moment from 'moment'
 import * as PropTypes from 'prop-types'
 import * as React from 'react'
+import styled, { css } from 'react-emotion'
```

Comment:
```suggestion
import styled from 'react-emotion'
```

import appears to be unused?

Reply:
yep, done